src/pages/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Layout from '../app/layout';
+import Button from '../components/Button';
+import Navbar from '../components/Navbar';
+import Testimonial from '../components/Testimonial';
+import Link from 'next/link';
+
+export default function HomePage() {
+    // We may want to get these testimonials from a static file or API
+  const testimonials = [
+    { quote: 'This app has been a game-changer for my studies.', author: 'Jason Parsons', title: '4th Year CS Student' },
+    { quote: 'This app has been a game-changer for my studies.', author: 'Jenny Ma', title: '4th Year CS Student' },
```

Comment:
Forgot a group member 🙂
```suggestion
    { quote: 'This app has been a game-changer for my studies.', author: 'Akshay Rash', title: '4th Year CS Student' },
```